salt/modules/boto_route53.py

```diff
@@ -659,27 +659,27 @@ def add_record(
     """
     Add a record to a zone.
 
-    CLI Example:
+     CLI Example:
```

Comment:
same here this entire doc was shifted and needs to be moved back to align with `"""`